CompuCell3D/core/pyinterface/PlayerPythonNew/CMakeLists.txt

```diff
@@ -132,14 +83,23 @@ endif(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
 SWIG_ADD_LIBRARY(PlayerPythonNew LANGUAGE python SOURCES PlayerPython.i)
 
 if(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
-  SET_TARGET_PROPERTIES(PlayerPythonNew PROPERTIES SUFFIX ".pyd") # changes dll name to pyd sop that it is compatible with new Python standard
+    SET_TARGET_PROPERTIES(PlayerPythonNew PROPERTIES SUFFIX ".pyd") # changes dll name to pyd sop that it is compatible with new Python standard
 endif(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
 
-SET_TARGET_PROPERTIES(PlayerPythonNew PROPERTIES OUTPUT_NAME  "PlayerPython")
+SET_TARGET_PROPERTIES(PlayerPythonNew PROPERTIES
+    OUTPUT_NAME "PlayerPython"
+    LINK_FLAGS ${undefined_link_flags}
+    SWIG_USE_TARGET_INCLUDE_DIRECTORIES TRUE
+)
+
+target_include_directories(PlayerPythonNew PUBLIC
```

Comment:
no need to do it now, but we could rename PlayerPythonNew to PlayerPython and remove the ancient code (PlayerPython) from our code base